src/client/stdlib/duckdb.js

```diff
@@ -182,6 +182,40 @@ Object.defineProperty(DuckDBClient.prototype, "dialect", {
   value: "duckdb"
 });
 
+async function installLocalExtensions(database) {
+  const repo = new URL("../../_npm/extensions.duckdb.org", import.meta.url).href;
+  const connection = await database.connect();
+  await connection.query(
+    [
+      // "arrow",
+      "autocomplete",
+      // "aws",
+      // "azure",
+      // "delta",
+      // "excel",
+      "fts",
+      // "httpfs",
+      // "iceberg",
+      "icu",
+      "inet",
+      // "jmalloc",
+      "json",
+      // "motherduck",
+      "parquet",
+      // "postgres_scanner",
+      "spatial",
+      "sqlite_scanner",
+      "substrait",
+      "tpcds",
+      "tpch",
+      "vss"
+    ]
+      .map((ext) => `INSTALL ${ext} FROM '${repo}';`)
```

Comment:
How do these paths get content-hashed and/or versioned (for immutable caching)?

Reply:
I believe they are versioned by the `1.1.1` in their path, but I'm not sure of what happens server-side in duckdb-land. I'm talking with @carlopi to understand this better.